backend/src/test/java/com/daedan/festabook/festival/service/FestivalServiceTest.java

```diff
@@ -109,9 +121,38 @@ class getFestivalByFestivalId {
                 s.assertThat(result.festivalName()).isEqualTo(festival.getFestivalName());
                 s.assertThat(result.startDate()).isEqualTo(festival.getStartDate());
                 s.assertThat(result.endDate()).isEqualTo(festival.getEndDate());
+                s.assertThat(result.lineups().responses().get(0).name()).isEqualTo(lineups.get(0).getName());
+                s.assertThat(result.lineups().responses().get(1).name()).isEqualTo(lineups.get(1).getName());
             });
         }
 
+        @Test
+        void 성공_라인업_날짜_오름차순_정렬() {
+            // given
+            Long festivalId = 1L;
+            Festival festival = FestivalFixture.create(festivalId);
+
+            Lineup lineup1 = LineupFixture.create(festival, "후유", LocalDateTime.of(2024, 5, 3, 12, 0, 0));
+            Lineup lineup2 = LineupFixture.create(festival, "미소", LocalDateTime.of(2024, 5, 1, 12, 0, 0));
+            Lineup lineup3 = LineupFixture.create(festival, "밀러", LocalDateTime.of(2024, 5, 2, 12, 0, 0));
+            Lineup lineup4 = LineupFixture.create(festival, "부기", LocalDateTime.of(2024, 5, 2, 11, 0, 0));
+
```

Comment:
시간 순서가 중구난방으로 되어 있어 읽기가 힘들어요.
오름차순 정렬을 기대하면
내림차순으로 값을 넣으면 개발자가 코드를 읽기 더 수월해질 것 같습니다.

```suggestion
            Lineup lineup1 = LineupFixture.create(festival, "후유", LocalDateTime.of(2024, 5, 3, 12, 0, 0));
            Lineup lineup2 = LineupFixture.create(festival, "미소", LocalDateTime.of(2024, 5, 2, 12, 0, 0));
            Lineup lineup3 = LineupFixture.create(festival, "밀러", LocalDateTime.of(2024, 5, 1, 12, 0, 0));
            Lineup lineup4 = LineupFixture.create(festival, "부기", LocalDateTime.of(2024, 5, 1, 11, 0, 0));
```